<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformanceTestDialog</class>
 <widget class="QDialog" name="PerformanceTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perform Scene Test</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabVideo">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="video">
      <attribute name="title">
       <string>Video Tracks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeWidget" name="videoTracksWidget">
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Width</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Height</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Timebase</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Aspect</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Field Order</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Format</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color Range</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color Space</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color Primaries</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color Transfer</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="addVTrackButton">
            <property name="text">
             <string>Add ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editVTrackButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Edit ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteVTrackButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="audio">
      <attribute name="title">
       <string>Audio Tracks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="audioTracksWidget">
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Layout</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sample Rate (in Hz)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Format</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="addATrackButton">
            <property name="text">
             <string>Add ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editATrackButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Edit ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteATrackButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Frames / Iterations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="frames">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
